packages/assets-controllers/src/NftController.test.ts

```diff
@@ -508,6 +529,63 @@ describe('NftController', () => {
       clock.restore();
     });
 
+    it('should add the NFT to the correct chainId/selectedAddress in state even if the user changes network and account before accepting the request', async function () {
```

Comment:
Hmm, this looks identical to the previous test. Is there something I'm missing, or would you mind explaining how this verifies that the correct chain ID and address are being used?

Reply:
Working on making this a real test right now!